apps/leave-app/backend/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "wso2"
+name = "leave_service"
+version = "1.0.0"
+distribution = "2201.9.4"
```

Comment:
Shall we use `2201.12.7` version(we can also use the latest ballerina version, 2201.13.1 as well)?